### ISSUE TITLE
Feature request: run tests in same tab instead of separate browser instances

### ISSUE THREAD

**Issue by simondib** [NimaSoroush/differencify#111]
Currently testing a responsive component or page layout across multiple breakpoints requires a separate headless browser instance. 

It might be much more efficient to resize the same browser window instead of spawning a new instance for each test. This would reduce test execution times dramatically.

**Comment by NimaSoroush** [comment]
window resize should not take that long. Can you run them in non-headless mode and see what is going on?

**Comment by NimaSoroush** [comment]
BTW, I am working on a new feature that will boost the test execution a lot (like 20 times faster). Still at POC level but planning to release it soon. So if the current performance does not bother you much you can wait for that

**Comment by simondib** [comment]
Non-headless mode opens a single browser, but launches each test in a new tab - so the load time is still lengthy. 

I tried removing .newPage() for subsequent tests to attempt to force the test in the same tab, but was getting strange test results.  For example, the first test passed showing correct styles, subsequent tests gave false positives (logo on the right should not be appearing):

    it("appear correctly on large devices", async () => {
      await differencify
        .init()
        .newPage()

        .setViewport(getViewport('large', VIEWPORT_HEIGHT))
        .goto(TEST_HOST + PATH)
        .waitFor(DELAY)

        .screenshot()
        .toMatchSnapshot()
        .close()
        .end();
    });

    it("appear correctly on medium devices", async () => {
      await differencify
        .init()
        // .newPage()

        .setViewport(getViewport('medium', VIEWPORT_HEIGHT))
        .goto(TEST_HOST + PATH)
        .waitFor(DELAY)

        .screenshot()
        .toMatchSnapshot()
        .close()
        .end();
    });

![image](https://user-images.githubusercontent.com/7483506/48586907-6b01c380-e985-11e8-9efa-291c8a271cea.png)

Ideally testing a simple viewport change could be run in the same tab. 

Great to hear about he POC speed boost. Loving Differencify, just looking at ways we can get the most out of it.
